pkg/apis/argoproj/v1alpha1/zz_generated.openapi.go

```diff
@@ -27,12 +27,12 @@ import (
 
```

Comment:
This is a revert of a change introduced in #313 which no one is sure where it came from.